src/components/singlEventPageComponents/EditEvent.jsx

```diff
@@ -16,39 +33,34 @@ function EditEvent() {
         // console.log(e.target.category.value)
         // console.log(e.target.location.value)
         if (id) {
-            const eventRef = doc(firestore, "events", id);
-            updateDoc(eventRef, {
+            const updatedInfo = {
                 title: e.target.title.value,
                 about: e.target.about.value,
-                date: e.target.date.value,
+                date: startDate.toLocaleDateString("en-GB"),
                 image: e.target.image.value,
-                location: e.target.location.value,
-            });
+                location: loca,
+                interests: selectedInterets.map((interest) => interest.value),
+            };
+
+            const eventRef = doc(firestore, "events", id);
+            updateDoc(eventRef, updatedInfo);
+            console.log("theeventREf", eventRef);
         }
     }
 
+    const handleLocationSelect = (selectedLocation) => {
+        setLoca(selectedLocation);
+    };
+
     return (
         <div className='bg-white border rounded-lg shadow relative m-10'>
             <div className='flex items-start justify-between p-5 border-b rounded-t '>
-                <h3 className='text-2xl font-semibold'>Edit Event</h3>
```

Comment:
this is the edit event no ?

Reply:
amazing insight i totally missed it, to be fixed asap